corehq/apps/app_manager/static/app_manager/json/commcare-profile-settings.yml

```diff
@@ -246,7 +246,7 @@
 
 - name: "Auto-Resize Images"
   description: "Automatically resizes images within forms. Follow 
-  <a href='https://confluence.dimagi.com/display/commcarepublic/Auto-Resize+Images+on+Android'>
+  <a href='https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143946731/Images+in+CommCare'>
```

Comment:
This looks like it could be more specific. The previous link pointed to resizing images, and I see that there's a section within the new link that would probably be nicer: https://dimagi.atlassian.net/wiki/spaces/commcarepublic/pages/2143946731/Images+in+CommCare#Auto-Resize-Images-on-Android